File: Documentation/6.0/Samples/python/ClientApi/Session/Querying/CountQueryResults.py
from ravendb.infrastructure.orders import Order

from examples_base import ExampleBase


class CountQueryResults(ExampleBase):
    def setUp(self):
        super().setUp()

    def test_can_use_count(self):
        with self.embedded_server.get_document_store("CountQueryResult") as store:
            with store.open_session() as session:
                # region count_3
                number_of_orders = (
                    session.advanced.document_query(object_type=Order).where_equals("ship_to.country", "UK").count()
                )
                # The query returns the NUMBER of orders shipped to UK (int)
                # endregion

File: Documentation/6.0/Samples/php/ClientApi/Session/Querying/CountQueryResults.php
<?php

use RavenDB\Samples\Infrastructure\DocumentStoreHolder;

class CountQueryResults
{
    public function samples(): void
    {
        $store = DocumentStoreHolder::getStore();
        try {
            $session = $store->openSession();
            try {
                # region count_3
                /** @var int $numberOfOrders */
                $numberOfOrders = $session->advanced()
                    ->documentQuery(Order::class)
                    ->whereEquals("ship_to.country", "UK")
                     // Calling 'Count' from Raven.Client.Documents.Session
                    ->Count();

                // The query returns the NUMBER of orders shipped to UK (int)
                # endregion

            } finally {
                $session->close();
            }

        } finally {
            $store->close();
        }
    }
}
